### PR TITLE
Change kismet in dungeon profits to bazaar

### PR DESCRIPTION
**Pull request by heyngra**
image

### CODE REVIEW

**Comment by efefury** [comment]
damn somebody uses this feature

**Comment by heyngra** [comment]
i had it enabled and -1 irritated me asf

**Comment by efefury** [comment]
> i had it enabled and -1 irritated me asf

i made the feature and i don't even use it myself

File: src/main/java/io/github/moulberry/notenoughupdates/listener/RenderListener.java [hunk 1]
```diff
@@ -777,7 +777,7 @@ private void renderDungKuudraChestOverlay(GuiScreen gui) {
 							}
 						}
 					}
-					double kismetPrice = neu.manager.auctionManager.getLowestBin("KISMET_FEATHER");
+					double kismetPrice = neu.manager.auctionManager.getBazaarInfo("KISMET_FEATHER").get("curr_buy").getAsFloat();
```
Comment (romangraef) [inline src/main/java/io/github/moulberry/notenoughupdates/listener/RenderListener.java:780]: `getBazaarInfo` returns `null` if the bazaar api could not be queried.

**Review by romangraef — CHANGES_REQUESTED** [review]

**Review by romangraef — APPROVED** [review]

**Review by NopoTheGamer — APPROVED** [review]

**Pull request merged**